src/config.erl

```diff
@@ -208,7 +208,7 @@ handle_call(all, _From, Config) ->
     {reply, Resp, Config};
 handle_call({set, Sec, Key, Val, Persist, Reason}, _From, Config) ->
     true = ets:insert(?MODULE, {{Sec, Key}, Val}),
-    couch_log:notice("~p: [~s] ~s set to ~s for reason ~p",
+    error_logger:info_msg("~p: [~s] ~s set to ~s for reason ~p",
```

Comment:
@elbrujohalcon it actually [ships](https://github.com/apache/couchdb-couch-log/) in this way.